tests/consensus_tests/docker-compose.yaml

```diff
@@ -23,5 +23,17 @@ services:
     ports:
       - "6433:6333"
       - "6434:6334"
-    command: bash -c "sleep 20 && ./qdrant --bootstrap 'http://qdrant_node_1:6335'"
+    command: bash -c "sleep 5 && ./qdrant --bootstrap 'http://qdrant_node_1:6335' --uri 'http://qdrant_node_follower:6335'"
```

Comment:
Did you intend to commit this change as well? It seems unrelated.

Reply:
no, it was for the debugging of the consensus

Reply:
can we still keep it?

Reply:
Sure :+1: